snap/gadget_test.go

```diff
@@ -45,6 +46,12 @@ defaults:
   core:
     something: true
 
+connect:
+  - snapid1:plg1  snapid2:slot
+  - snapid3:process-control
+  - snapid4:pctl4 :process-control
+  - snapid5:pctl5 system:process-control
```

Comment:
The values here look a bit curious, at least in a first impression. Perhaps not bad. Do we have precedence for this, other than CLI?

Reply:
yes, it's a subset of what is supported by the CLI, we can chat about it